native/packages/react-native-bpk-component-card/readme.md

```diff
@@ -0,0 +1,57 @@
+# react-native-bpk-component-card
+
+> Backpack React Native card component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-card --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import BpkCard from 'react-native-bpk-component-card';
+import BpkText from 'react-native-bpk-component-text';
+import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: TOKENS.spacingBase,
+  }
+});
+
+export default class App extends Component {
+  render() {
+    const copy = (
+      <View>
+        <BpkText textStyle="xxl">Flights to Edinburgh</BpkText>
+        <BpkText textStyle="xl">Flights to Edinburgh</BpkText>
+        <BpkText textStyle="lg">Flights to Edinburgh</BpkText>
+        <BpkText textStyle="base">Flights to Edinburgh</BpkText>
+        <BpkText textStyle="sm">Flights to Edinburgh</BpkText>
+        <BpkText textStyle="xs">Flights to Edinburgh</BpkText>
+      </View>
+    );
+    return (
+      <View style={styles.container}>
+        <BpkCard onPress={() => { } accessibilityLabel="Example Card"}>{copy}</BpkCard>
```

Comment:
There's a typo here, the closing `}` for the `onPress` prop is at the end of `Example Card`. It should be:

```<BpkCard onPress={() => {}} accessibilityLabel="Example Card">{copy}</BpkCard>```

Reply:
Isn't your convention to just return null in example callbacks. i.e `onPress={() => null}`?